capstone/src/test/java/com/google/sps/servlets/student/StudentServletTest.java

```diff
@@ -304,4 +313,45 @@ private String getAnnouncement(String announcementContent) {
             time);
     return fullAnnouncement;
   }
+
+  @Test
+  public void doPost_studentClicksLeaveButton() throws ServletException, IOException {
+    studentMegan.setProperty(Constants.PROPERTY_CLUBS, ImmutableList.of(CLUB_1, CLUB_2));
+    datastore.put(studentMegan);
+    datastore.put(club1);
+    localHelper.setEnvEmail(MEGAN_EMAIL).setEnvAuthDomain("google.com").setEnvIsLoggedIn(true);
+    when(request.getParameter(Constants.LEAVE_CLUB_PROP)).thenReturn(CLUB_1);
+
+    String responseStr = doPost_studentServletResponse();
+
+    // Access local Datastore to get student's new club list
+    Query query = new Query(MEGAN_EMAIL);
+    PreparedQuery results = datastore.prepare(query);
+    Entity student = ImmutableList.copyOf(results.asIterable()).get(0);
```

Comment:
nit: I should've caught this earlier, but it's probably good to assert that  results isn't empty before calling .get(0). Same for all the other tests

Reply:
Done! Thanks for pointing this out